utils/py_es_bulk.py

```diff
@@ -149,19 +148,26 @@ def actions_tracking_closure(cl_actions):
     # Create the generator that closes over the external generator, "actions"
     generator = actions_tracking_closure(actions)
 
-    streaming_bulk_generator = helpers.streaming_bulk(es, generator,
-                                                      raise_on_error=False,
-                                                      raise_on_exception=False,
-                                                      request_timeout=_request_timeout)
+    streaming_bulk_generator = helpers.parallel_bulk(es,
```

Comment:
can you please make this optional to use? following discussion on #185 ?

Reply:
made optional, defaulted to stream indexing